osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -38,6 +38,7 @@ private void load(OsuConfigManager config)
                 {
                     LabelText = "Show health display even when you can't fail",
                     Bindable = config.GetBindable<bool>(OsuSetting.ShowHealthDisplayWhenCantFail),
+                    Keywords = new[] { "bar" }
```

Comment:
I think this should be capitalized as "Bar". It doesn't matter, but just for consistency.